### PR TITLE
Remove "scroll" option from security:searchRoles doc

### PR DESCRIPTION
**Pull request by Yoann-Abbes**
## What does this PR do?

Remove "scroll" option from security:searchRoles doc

## Other change

Change script test name for admin console tests
depends on https://github.com/kuzzleio/kuzzle-admin-console/pull/829

### CODE REVIEW

**Comment by scottinet** [comment]
Shouldn't this PR target 7-dev?

**Comment by Yoann-Abbes** [comment]
> Shouldn't this PR target 7-dev?

I thought I wouldnt be a problem to merge it directly on master to fix this error as soon as possible.
What do you think?

**Comment by scottinet** [comment]
I don't see why this would be an emergency: the method isn't event documented.

**Comment by Yoann-Abbes** [comment]
> I don't see why this would be an emergency: the method isn't event documented.

Base branch changed to 7-dev

**Comment by scottinet** [comment]
Uh... so, I finally read the title, description and code of this PR. I thought this was about removing some code, not a documented option.

I apologize. And this PR, of course, should be merged on master. :expressionless:

**Review by scottinet — APPROVED** [review]

File: .travis.yml [hunk 1]
```diff
@@ -97,7 +97,7 @@ jobs:
           npm ci
           npm install kuzzle-sdk-v7@$TRAVIS_BUILD_DIR
           cd test/e2e/run-test && npm ci && cd -
-          CYPRESS_RETRIES=5 npm run test:e2e
```
Comment (Aschen) [inline .travis.yml:100]: End 2 end tests must be run with the `test:e2e` command. You should change the `package.json` if it's not the case

**Review by Aschen — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (Yoann-Abbes) [reply .travis.yml:100]: That means a PR on admin console repo and merge to master https://github.com/kuzzleio/kuzzle-admin-console/pull/829

**Pull request merged**